### PR TITLE
Fix shibance

### PR DESCRIPTION
**Pull request by lljxx1**

### CODE REVIEW

File: env.sample [hunk 1]
```diff
@@ -0,0 +1,17 @@
+ETHEREUM_RPC=https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161
```
Comment (0xngmi) [inline env.sample:1]: You might want to revoke this api key

Comment (lljxx1) [reply env.sample:1]: That was taken from MetaMask

Comment (lljxx1) [reply env.sample:1]: > You might want to revoke this api key
That was taken from MetaMask

Comment (lljxx1) [reply env.sample:1]: That was taken from MetaMask

**Comment by lljxx1** [comment]
fix uniswapv3

**Comment by 0xngmi** [comment]
I'd prefer to avoid the use of hardcoded pool lists since these will go stale really quickly and maintaining them is a pain

**Comment by 0xngmi** [comment]
Also, not sure if for now we should stick to the old version of the adapter since the one on this PR only has bsc

**Comment by lljxx1** [comment]
> I'd prefer to avoid the use of hardcoded pool lists since these will go stale really quickly and maintaining them is a pain

Yes, maintain is the big problem of this kind of product！

**Pull request merged**